Account/FxALoginStateMachine.swift

```diff
@@ -115,6 +115,9 @@ class FxALoginStateMachine {
                                 log.error("Unknown error: \(error.description).  Transitioning to Separated.")
                                 return separated
                             }
+                        case let .Local(localError) where localError.domain == NSURLErrorDomain:
+                            log.warning("Local networking error: \(result.failureValue!).  Assuming transient and not transitioning.")
```

Comment:
It does: `result.failureValue` is an `FxAClientError`, which is verbose in the case of `.Local`.  (See the existing logging that appears in your bug report.)